### PR TITLE
Implement logic to make filters functional on the product  inventory list page ( #232z8jk ).

### PR DESCRIPTION
**Pull request by meet-aniket**
This Repo depends on **PR no. 72**
Link: https://github.com/hotwax/commerce-hub/pull/72

Reference: 1. 
image

Reference: 2.
image

### CODE REVIEW

File: changelogs/unreleased/-232z8jk.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: 'Implement logic to make filters functional on product inventory list page.'
+ticket_id: "#232z8jk"
+merge_request:
```
Comment (ymaheshwari1) [inline changelogs/unreleased/-232z8jk.yml]: Add merge request number.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/product/actions.ts [hunk 2]
```diff
@@ -93,6 +93,79 @@ const actions: ActionTree<ProductState, RootState> = {
     }
   },
 
+  // Update QueryString
+  async updateQueryString({ commit }, queryString) {
```
Comment (adityasharma7) [inline src/store/modules/product/actions.ts]: I think this is not needed. We could use updateProductFilters

File: src/store/modules/product/actions.ts [hunk 3]
```diff
@@ -93,6 +93,79 @@ const actions: ActionTree<ProductState, RootState> = {
...
+    commit(types.PRODUCT_FILTERS_CURRENT_UPDATED, { 'filterName': 'queryString', 'value': queryString })
+  },
+
+  // Update Query
+  async updateQuery({ state, dispatch }, params) {
+    await dispatch('updateQueryString', params.queryString)
```
Comment (adityasharma7) [inline src/store/modules/product/actions.ts]: I think this will reinitialise it to undefined for the updateProductFilters action

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/store/modules/product/actions.ts (hunk 2 above)
Comment (meet-aniket) [reply src/store/modules/product/actions.ts]: Done Sir.

File: src/store/modules/product/actions.ts (hunk 3 above)
Comment (meet-aniket) [reply src/store/modules/product/actions.ts]: Done Sir.

**Comment by Dhiraj1405** [comment]
Not priority for now. Hence, closing this pull request.

**Pull request closed without merging**